src/Components/VideoSession.jsx

```diff
@@ -47,7 +50,7 @@ export default function VideoSession() {
       console.error("Error fetching session and token:", error);
     }
   };
-
+  console.log(fetchSessionAndToken)
```

Comment:
do you need these console.logs in your deployed app?
If this was only for local debugging you can remove them before merging;

Reply:
We don't need the console logs anymore I'll work on removing them for the deployed app